optuna/samplers/_tpe/sampler.py

```diff
@@ -205,6 +205,10 @@ def objective(trial):
                 and the durations of the running states are significant, and/or the number of
                 workers is high.
 
+            .. note::
+                This feature can be used for only single-objective optimization; this argument is
+                ignored for multi-objective optimization.
+
```

Comment:
Alternatively, remove this note and update `constant_liar`'s explanation as follows since `constant_liar` already has three `note` section and added section looks short:

```
            If :obj:`True`, penalize running trials to avoid suggesting parameter configurations
            nearby for only single-objective optimization. This argument is ignored for multi-objective optimization.
```